_extras/checkout.md

```diff
@@ -174,7 +174,7 @@ meetings, CarpentryCon, and CarpentryConnect events are all eligible to satisfy
 
 ### Getting Credit
 After each Community Discussion meeting, hosts submit a form that includes the names of attendees who identified themselves as
-fulfilling their checkout requirement. This is indicated by writing "checkout" next to your name in the Etherpad. As long as you
+fulfilling their checkout requirement. This is indicated by writing "checkout" next to your name when you register in the Community Discussion Etherpad. As long as you
```

Comment:
```suggestion
fulfilling their checkout requirement. This is indicated by writing "checkout" next to your name in the Community Discussion Etherpad when you register. As long as you
```
swapping order of clauses for readability

Reply:
This change is unequivocal and can be accepted.